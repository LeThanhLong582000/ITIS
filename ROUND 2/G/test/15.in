57
11111110101001110101110111111101011000001000101110101111010000101010000101001000001011100001111000000001111110101111101111011001101100100011100011110000100110011000
10111001010011101001010001010001101110111100010011111111000111101101101010111111000110000100111011010111100010000101011000101000010101010101011011001000000000100011111111100000011101111000111011001110000000111110010110000001100011110110110101110010110001100100100101010000101001010101111111001100101110100110011101110110111001110100111111000000111101110101111101111101011100001010000010110011011010010010011011010001101111110100100110001101111101110
01001111100100001110111101011011101010000100101101101100010100011010101100010100100110110111011011011011011110000010110010101000001010101110001100001110110001001101111011000000111101101100000101110001011011010010001111010000101100011101010001001011001110010010100101100100111001101001100011111011010110011001110000101111010110110011010010100001011111001110101000101101001101111010010011001111101101111011100101001110111000000000001100010110001100000100011101101101101100001000110110011101000010010100111010010010101101100010101101111110101001011111001101010100101010000011110100001101101000000111111000110111001010011100000111110011110100111000100101011110111110010010000110110101111111000000010101110110001010001111110101010101101111110110100001011001000111111111111100001101111000011001100111000011011011100
110110110101100010111000110111101000010101010010110110010000001011011111110010000000101000111110111110000011101100010001000100101100101111001001011110111101011000100000001110111110011101101100101000000101101010101001100101100101111100001100111100101001001101000110010111011010011100010000110010001100100010010010001011100001111101101101110011010001000100110000000101010011001111110010100111001111011011001101000011010001000111011010000000110001000101001001111110001101010110010100100111100010101001011110000110001010010111011011100011011011101010011010000010011010111101011101001100010011001110111101100111011100001001001101001101011011111
11010001010100010000001100101111010010110110011101000110010001001111101101101111101011101110011100100111000000001100001001
001111001101110010101011011010000001100100010001101110111100101101000100100110111010100111101010100101111110111011001101111100110000111101101101011110000111001111100101010000010011111010110100001110111010100010001110100100000011110001100010111010100011101111100000011101011001111110110110101100111100111001100000100011001001001001000101001100101110011010000100100101001011101010101011110010010001000100100010110001011001000010101011110011110001011111011111111100101100011000011010001010010110110001101010100010010110110010011010101101000011011100010010010001101011111110000011101100101111001001110101110111100010011011110110000100010010011111101111101111110101100101010101110111000100010010000011010110110000000110101111000100011000001011010111111101110011100101101011001110111010100011111110011101101011001110100010110011000010100111000000010100101101001010010100110110101111100010001011010000111010110111011001100010101111000011110101000111001000100001111110100001111000111011
0100010000111110001011011110101100111110110111010110000110011100011101100000011010111110111001000000000100011101011001100000100010100010101110101011010100000010110010110001001000001001111110000000001110100000000100000010000100000101110011010111010000101101010010000010110111011111000001010000101010011011110000111010
0000000110000000111001101001010101001111111000111101001000000110100011101001110100010101111000011110000100010001010011010011000001100111100111010100101101111011011110010111111110101011011011000011111111111011010101011010101001010110010101011010010011001110010010011110001101111010000111001011011000010011011000000110110000101110000100101111111000100110000011000100100111011100001000110101111010101000010001110111110110111100111011010110001110000101011100001110000000110110101100110010011001010010000100000100111011001011000001010100010001011011100011111100011001101001000011010111111001101111101110011010100000100111101011101000010100100010101001011111011000100000010100
0101001010010101011000010100101111101001111001100001101100010000001000001110101111110101101110001011111110101100111101011000001001110000110001100010010111010111111011010101110001000011111011101111001101000000011000101001100100101110001011001101110001010010001000111001100010100111111110101011001000101001110011010011110111101111111111000011101100000110100001000101001000011010011101100100001000000111110111110111111101100100011001001010010111001010111101000011010000110111001100100011011000011010111100101000000011010100011010011000001100111010001010010100111110111001001011110100110000000100101011001101011010100001110001110101011000010111100101011001100100001011000111110100110110111011010010011010110111000011000011001110101110101000101101100001101111101100000111101001010001010001000101110111001000110000000011100011001101001100110110011011100001010000101110101000111101110110000111111101100100100001101111001011010100100010000100011110100111100001011000
101100111001111110000001000110100000010001010100001111110101001100011001011110001100110101001111000101000001001010011000011101011010011000100110101001101110000011101100100101011000010011001011101000010100110100000100101011110011010010010111101001000001010010001111011011100011110110010110101001011110110000111011000010000010010100101110111110100011011100111010111101000001000110110001101010010100110001111101100111111100101000010000110100000111010100100000010000000101000101000101001001101011100100110010001010000101001100110110101000111011000100000101110001101110001111101110110100000011100000100001110011110001100011111001111101110010010001011011110110110011111110110011100001111001010001000100110111010010100010111001000001110110111101111011000011001010010010001001000111000000011011011101110110010010011111000011011100101001010000111001011101000110101011001101110011000111
100110101010001000011000101100000000000000101011011100011101101010000100111110001110111000011110100111011110111101000100000110010100101001101101011101000100011110000100111101101010100011010100101101111011100010110111111011110000110100111001111001011011111010111
11101111010111111010110011000100010001101011000101010111110000000110110101111110111011000111101100001001000111000110101001011100001001100100110001111010100000001110110001100100011101100110100001011010111111111111111000001001010000001100100100010001001101100001001011111111
0000000010011010110100111010010111101100110111001101010010001010000100111111110010100010111101110100001110001101000100010011101010001101110101110000000011000000011011100000101011110000111000100001110100000100010101011
00011110000111111101000110001010110100110100101010111010011011010001011001101010111001010001001011010000011001011001001011111011000000011001010100011000000100100111001011011010000100000111000110100101101110111111011101101111100001100011110111000101101100000101111000100011101001000000010100001101011101100010100011011111111101011111100110011011011000011011111100011111100100001110111101000100010110110110111101001011010111111000100000010000100111001111111100001100000000011000010110010111010010010111101001110000001110101111001011000010111011100011000011100100111000111100001100110001110101101110101000111100100011000100111010001011111100010100110101111010101000110100111101001110001100110100111101110
10001111100010010010010000100010000011011000010111010111001011011010010011101011001010100111100010100011110010110011100100110001111101110001100111000101101101001010001101010000011001011101101100100101100010101110100001001011111011010100011001000101101011010111111101010001100010000110010010001111000110111101001001010011101011100110001110001001000001100111000111011101010111111111100010110010000111010111011011101001000001011100000000111110101000110111101111111110001001111101100101111011100011110000010111101110111100110100101011110010011011110001010111011001110100000110101101100001010001110011100011011110111110110110111
000011100100110110001110010011011101101101010011011011110100000100101001100000111011110011100011000111101100110000101101101011100100111010101100000100000000100001110111001010101001010010010010111100100100101001110011000110
0100101101110001100101011010110000000010111000111001100100110111001000011001011100000000100000010000100010010010001000001010010111001100011001001010110100000011110101001001100001001100110101001001100110000001000011000001001110011111011101001110010100001101111001111010111111010001011111111100110011100111010001001110111001101100110110110001000111111001010010011000010111010010001001101010001100011010101100110000011010100111100010000001001110001011110001001010011110010101010011101110011111100010111001101000010001001101011010011001010001010000011011010011111101101010110110010101101001101011111111101001101001011101100011100111011101101011100000101100000000011111010101110100111100111011000011110010111011110111000001110011010000110010111010101111101110010010000010
10101100010101000000011100101111011100111101001000100000100011111110001101101111101011001010010010100110000100100001011000010001001011000100001010101011111010010011011100110010111001010010010000001111000100110110111101001011000110001110111100111111011010101011011000010011011100001010100001011101000101111100011100011011010111100110010011010011110010001011001011111011101101101101101100000011110110101100100101110111100001111100000000011001111100001010100011000000000101100011010100111011001101011010100000110001000010011110001101000011000100000
0010010011111101110101111110010011110100000101000000000011101000000011110011100010010100111101001110010011101010110001001010011000111101111100011011100000110101000101000110000101111101011001011000011100001000111011001001010001111100101101001010111100100100001110010
01111100010101010001111010110010100110000010011101000110001010101101010101011111000000000000011001001100001101000000111100010100011000101010101010011100111101001111010010101100101000101001111111100011001010100101111001001010100010011010111001001110011101011101010110111101010111111011110000000101100010011111011001001011111011001101100000001011101111010011000101110001010101111101110000110001100101101011001001000010011001010110111011011100011000011100001001101101111110011100001
000101100111001110001001111111101110001001000110100110101011111011111101110110100000100100001001010100010100111101101011101100101001001011000011001111011111111011000111010010110011001001010010111011111100100011010110100100000101011000110100111111001010110110011010010001000001100100011001001001010100011000110101010111100110110101001101101011010011100001110000100011111100010111000000011010010001011111000000000010101011000010101110010100011010111001
00110100100101010000000110010000101000110111111000101110100010101100110110010101110011010000011000110110000111101010100101110000101101101011001111110001000111000111
11000101110010111010011000101011001101001000100110000111011011110110001010111111010100000011111101011110001111110000111001010001010000110111110001011100010001111011011111011000000010111100101010100000101000101100011100011101101110100111100001011000111000000100100111001001011110101010010101011001110110000100010111001001000110000110110001101010100111111010010011101111111101000011000010010001101110001001101111100110001111011110000010011011110100110011010001011000010100100011010101101101010111001001111011000101
1111011111111100001100101111111100111001111110100110011111000000111100101011101010101010100011110100000011110101001000010011100100001011110010001101110111011101001011000111100011011111111110010101110001110101100011011100111110101100111001000111011111101100
1101011101110001000101000101100000001010000100101100111000111111011000001111111100000101000111111111011111110110011000110110111010110001111010010101100000100100100000100011
011110111110101111010100111000101001010100011010111001011000011100110111100010110000100101111110011100100100000101111011111001101000110000110001000110111111100000000110100101001011100111001000010100110011011100010111001101010000010100010101000010001100010110111110011111110100000111110000110110101011000000010110011000111100011111000010111100001111110110010100001110111001010010001010010010011110010100000111110101001111111100110010010111001000101100001001110110111101110010011011011100111010001011101101011110100101110001110111001101101000000110000110110110010100100000110100010000111110100111111010101111010001001111010010110000101000000011101010100010000011111011000010110000000101101111000011000001010011111110001101101011111010011001110110000011100110000010100010000001111001111010100001010000111000011110100011110111101001000111001101111000110011010010000101011010100100111110110111001100011000110101001010011001000111101011101001
1111101010100101110101111001011011111111001110100111110100101101011000000100000011101000010011110110001001010100110111100110011010111001000111010101001010111011011110111100100010000001001000111010010011000100100100111000010001000111111001111000101010001110
000101111010110001001001110010111101010111010110001000010111110011110100001101111011011010011101101110100101101111110100100100000111001011110011000001001110110100000100110011101101101010010010110001111110110001000001111010000101010000110101100111111001100011100010000010000100011000100010110110000110100111001100000110101111001001001110001011100010100111110111001110111010000110001101001101101110100011101101010010101110100000101001000110010100100111100011010000110010011001110010111101000010101000010011000011010001011101000101110111101000101001010111110000010111001000000011100101101101000001110100101011000101111101111101111111010110011001110110011010100
1100011110000110110110010000011000111100010010100000101111110000111010100010011001010010011000000011111010101010111000011101101001101100101011110101001110100110001001101110010101010100111111011011001100010011001111110111110101001100000000010001111011100001101010110101011111101000011110100110000101111100111001111000001011011010111110010111011010100111110010110110101110000101101001101101100110010011011110011011010000001110111100000001100001110101111101000110001011000101010110110001011001011010101100011100110011011111010100101000010110010111111111100011111001011000100101111110111100101111001001110101000100000110100100011000110011001001001100110010000000000010111111100100111101001010100100100001010011011110110011000101111011101110100011110100001100101011101111011000110000100011101101001101111010011101111101011100110111101011100001101001011001111100
00000111000100100011000111111011101000110110100100111001000010110110011001011111011101100110011111101010001111101001101011100111110000101110011101110100110000010100000001011100100100001001001111111000001110101010000110100
111000100100011101001011101010111101100011001110000011010011111101001010001101010010011000100101111010100100011011000110011111110101010100111011100111111000100101101111011111
1011101000001011010101001001100010011010010110000011000100110001001000100111101111111101100011110110110110101011000101111101111101010111110000000100011010001001110011111011000011000000110001001001101010000000110000011000111001100110011010001010000100100101010111000001101100101110111000011100011101101010101011010101101100110100011001101110011101110111100100010010111100111110000000101101011111110111010011001101110101011100101010011110110010111000110010010111001111001000000111111111111110000011110000001110010010101
011010111001100101110010000010110010001110010111001011010000001011101000110100001100100110011011000100011111110011100111111010000010110100001000111010010100001110110110001101111001000010010011010100011101000111000010011100011110111001011101110011001011101110100100010010001010011010101101010100000011111010001101111101101010010101011000000110000101010011101111101111100111110011001100001111011010000110011110100001011001001110100010000110010110001001011110001101011000010101011011101010011011110011110000010010011001000100111000110101100001001001000100011000111000111101001110110000110000111000011100010010001110011000000101011001000001110111100011011001101010110010101111101111010000110010011110000111101000111001010001011100101011000010000110010110101110111000001011111001011101001000010001000010110101110011111010000001011110110100010110000001011110110001001001010010100011010000110110101001010000101011110100111011110110001110110101010010110001011011
01001111101100101101111010000011011101101111011010011000000111110011000001110111111010000100001110000100001110001000100111110110110101101111011001000010000101000110101101001001101111011001011011101000110001111001111001110000101011101101000100111111101010000111001001010010000011110100110001100110100001110010101010001001101000000110001000111010110100010000100000100110011110100100111011110111100101000000111111111100111100101000011100110011110110011000001010010111001010001100001101111100000001101101001010001100011110011001011001001000100000110100011101110100
11111101010111011101010100100001010101110010110001001001010001001010110110110000101100010010011111011101110001110010111000000110101000101110110010011111000100111111100101111001010000110000100000110110101110101010111101011011001101100101100000010100110101001010110100010001011101110100011110010011011000010110110101100101111011011101111011100100010111101010111011100100100101101100110101011001000100110111111011110001000001110011110111011000011111110101111110111110001110000001110110001010110110011111000000100111100000011011001010110010001001011111001001111010001101100111110111001010000000100010011010001110011100101110111000111011011000111101101100010000000010110111011100111011010000010101111001111001100101011101100100101000010100111111111011010101011001010110010111110011111101001010101011101010011010000100010010101001011011011010001010000100001001010101000100011010010100001100110101110010110001100010000111100000001011101110000010000
110010000011000101011100001001100110111101100110101011101011100001000100001111100101011100101110011011101110111001101011011100111010100000111100100100100010110010011100001111101010010111000001100000111110010110000001000010000010010101010111001100000010110101101100000110011110110011101111001000000010100010110001001010101101100101100000011000110011001101100001001100010011110100010100101111111110111011010011100010001000111000000000010100010111100011111001001001100110000101100111001100101010011100110011001010110110111000010101001101011101
101010101001000011111011011010110010100111111110110100011111110010000011000110010000110101010110000000100110001000100011011000010001101110010010100000110110011111100001100100011001010001010000101011010110010101010011010110101011110110100101111000011001101010111101010000001100100010110000000010110110010000101010100111001101100010101111110011010001100100010111101111000100001000011000100111110111100010111101000001110010001100011110111101001111011110111101010100111011000001011101100101010011011111110100101010111111
0100000000001101100110001100010100110101101001100111100000111110001111110110110100010110010111000100111110000001101001000110011001000001111100001110110110111111011001100000011010100010001100110110010011110110101001011110111100010001000111101110000101100100001011001100010001111111001111010111001100110110000101110111011010011100010110101010111100001011110011101010001010001011110011000011011100011111011101101011010001000000111101111110100100111110000100100001011000100101100111100010001001001101110001111110001101101100110010101110001110100010101001100101001111111111000101111011110100101011101001001000101111011110110101001101101011010010010000110000111010100011000010100101111001010010010001001010000011000111010000010111101111011000001011110100001110
000100001010010000011011100101101111101110100101111000011111001011000011110010010100111001101000101000101101011100111101100000110011100000001111110101100111101011010100010010101000100000001111100010111011001110111001100010100111011000111110011010100010100001101000110011001011101001000000000010110011101011010110100111101011111101001010101000011000100001011110110101101001101100001111110111110100011101111001111101101111011111110110000000101101111110101010111101010111010001100000001110110100111011000101100001100110011110010000110101100101011001011111110001110011011110110110101101110001010001001000001010011011101011111101001000000110100001011100010000000110101010010101110011010110111000001111100
01010111100110001110101001011011011110110111110010100111111101001101100101010000011000100100000100000010011000010100000010100110110110011001011011101101011010101001010110111011000001010010011100101110000111101110100101100101111010100110010011000110111101100010110101000010101100010001011011011011100011101000111101110101100101100001110000111011000110010000101000001101010011100111100011011100001001000011101011100011011001100101001
00000001001010110000010110111011101101011100001001001100011010111101101111000111110001011011000100111000111101000101011010001111000001001001110001001101101000110111011001101010001100100100001101011100011101010010110011011010100011010111011111111000100010010101110101010011010011101000011101011000100011111110000101001111101101000100111010101010011010101100111110010001101010100101101010101111100010101100110101000010010011001101001001001011110000101110011011100011100100111111000010101100011100000111010010011101110101010101000001000001100001010010100100010101000100001011011111100110010100100111111000011111111011010110011010111001011100010011010110110101111101111111010100000001011010011000010111001000111011101111100001000110101011101011000111001000010001000110010010011000110101000111000010110110010111100000111110110101010011101111101011100111100101011101111010110001100000100001000000110011110001100000100101101111010100101000011011111100110101010010101110110001010110000110000011
01110000010001110101100110011011000110011111000001110100101010000111111100100110011101010111110111110001001010100001011010101111001001100010111000010111001010001101011000011001011110111011000101110100101011111011010111100010000010001110100000001111101
0001000001001010110110001100101110010011011011001001001011101111011001000010110011010011110010111010111111000110010111011000100010001010100011111110001001100011001000011010111100100110000011000001101011001111110110000001011111001001100010010110011010101001110101010111110110111101100001100110100000011001101010100100001000000100011110001001010100000010010010001011101111000011100101010011101110011010111001110000000010
000110000000100110001000001001100111101111010010100010111100011110100011100100101011011000011001010101100101100011010011111100100010101111001101111001000011111011100001100110100111001110010111011011011000101000101100001101110000111110110000100001100101010011100011000001000011111100011101000111001101010000000000111001100010110101101111111010101011111110110010011011110010110111011101010100011000100
1110100001011000010111001010011100110011101010111011011111000110010000000100010001111111100011011101110001000111010110001111001011001100101111001000000000101000001100000011111001010010011111001111011111101110001100101000100001110111011011100101101110100001011010010110010111011000100110011100000000101010000100001111101111000011111111011100001001001001010110011110111000100110110101010100011011100011010000000110110100001111010010100110000110010110101100001001010111110010011001011011101010111101000100100100011111010101101101010110000111100110001000110101011000001010000011011110111011001110011110111110111001011110111101000100111010111010000101011110111000111100000100001011001101010000010101001001011000100011100111001101000010111110110100100000000010101011110000011101010001101111001
011010011101011001100100111110111010000110011010000110010010010101001010000101000101100001111110101001111110011011100111111101011001011101100011111101010101101111111010110000110110110100101011010110101010111000011010000110000111010011010001011110000111110101100101101111110111110011111000001010101011001011001100011100011101100100100011000110101101011101011011010001011111101011010001001001000000000100000001011101101011011111001100111111010111100
101101101110100000110011010110010111100000101011100100010111000100011110111011000010110100111111100010110110000011000010011010100101110001101100100111110010010110110110010011101100000110100111100111010111010101011110001010001100110111100001001001111001000010100100001100110000001110010001011011001100110010001110111011011111001100010110010101001101010100100110011101100110010111011110001100001111110011110100011000000100100000001010011100001101001001010010011111110110110100101111100000100
000001010111101000011111000101110110010011010100110010111001110011100100011000101111011110011101001001101010001000011110001000101100010110111101011111001100001010111110011100110101010110110011011010101001001000000101110111010001100111100101100110001010111111100111101101101010000000001001010010011111010110010111111111111110001100011000010101111011111010010101011011001100101100100100111101010010100100000001010100110001101011110111111000101001101011011100101000010101101110100101010011100001010000101000101010100010101110110001010011110000111110000000101110111100100111110011000100111001010100001001101100011010001000101101010110001000000000110101001100010010000111001101001010111100100010100110101101000111010011111101000111011100111011101000000101010101111000110110010101100110100100101011101110111101011110001010110011001000001110111010010110101110110100100000001011101011010001010100000
00101011100011110111100000100111100101111100010011000001110100000001110000000111110010100110100110101110111111000001101010001110100010110110010011100110100001100000110000101101010000100110111000000010011001111111010101011101101111111101011110111010010110001101010100110101100110010001011010110111010011010110111110001111000011000000001111101111001110001001011111101011000000100010001101000101010100110101101001011111011111011111100000000111000011110111001010110110000010101011101111111110110000011110111011101100100010000101001001011011010011100101110101011111100101001110111100011010011001011100010111101111101101111010010000101100100100011110101001000111010011111011000100100010010111100111010100111100101110101110011110111111101010101100011111111100010001100111010101110100001000000000101011001000110001100011101101100100001110100000111111011000101000011110111111101000
0101101101001001010011011110011110100001101100110000111001011010011011011100111100000001100010110010011001101001101110011101001101011011001100010011100101111111001100011100101010011010011010010011111001010100011000111110111100000100100100011000100011100001101010100001011110001111111010110111011011011100110001100110010110001010110010100111111000111001011010110100010101110110100000001111111011010011101000110100001000100001010111111100010011100110101110111010001011001001011101010011010101000110011001011111011111001010001101011101101101011001010010010000111010000000011100111011101000010100110000001001101001110111110100001010001100001100111111010110000001111100101110011100001111111001
1000001100101111110110111110101000100110010111100011011100111101100110011111111111100111010110001011110101010001000100001101001110101110100000000110111000011001000001100001101011001000001000100001001101110111110110110111011011000101011101010001010010011010001010100110001111001001011001010100111011001100111111011010011100100011000010001100111110110111100001111101011111010111001001000100010000110000010101000000111100000101000100111100100100101101100110011010100010011111010110011111000111111111001101100001101101100011010100001010100011110100101110111010101001100000111001111101110001111000110000101010110001000011101100111100010110111100110111011001111011100111011011000100010111101101110110101110101101111011011100111111101000001011110000101101110010101111001110100001111010001000100000011000111111101111011001100
011011110001100111110011000101010111001000100100011001010011000001011101000001110000111100100101000010000001101011110001010111110111000001100110110100000011011100010001010101111000000
010010101101111111100111111101100011110111111011011111100100011100010001011101001001110111101100110111100000100101100101011100010110010100001101001011010110001110101000010110111000010111000011011110100000010001100011100101110011010001010101010010101000000111001010111100111111011000101010000011001101011000100100000110100110101000010111100100
1111110010010100101001111111110101110001001000010011100000011000110010101000100000001100100100001100110010011000001011100000110101001000011001001010010111001010011011101010011100010000011011110100111001001100110010101111111001000000000001100101011011001011010111111000000110010011000100100011101010010010011001011101100111100111001101101101011010110000011110011101000101011101001111000101110110001110000111010010001000110000111011010101010110100000100111010101001111111010110110100110010000110001101111110101111011101101011011100010001010000001110001001001001111111111000101010000111000111000001011011110110000110110100000011100110000001111011010101100101001111010010000110100111010100001010010100101000110110011111001100100001111000101100101110001001000100010011110110101001000111010010010111001000010010001001111001100011010001101000100110000011001000111101
101100100010100011010010010100000000001110010011111100101100101100011011001111111101111100000000110110001111000010101100011111001011011001010011111011010010001010111001111001100101101001110111111110001101011000011010000000101111001100011100001000110000101001011001011101101000001111101100101010001010101011100001000101100010011110001011100010111010011100101000011110010111000100111000010010110101010111111001010111010110101111101110000001111100100011110111001011000000010100011001110110100001101101010011000001010010100111101111110110110010101111001100111000100011010100111000110101111100111100101111100110
111001111100010001011101110010111000111111100000100001011101001010011011110001000101001001000111010010011011100110101001000000011100000100001010110100010010110101111100100110111101000000010000100100011000110111001001100010011100110001010111001101001010000011110110110001101110111
0110000110001101000110110100111101011001011110010011111000010111011101100101001000101101101010001011100111000001011110000100111000101010010011011001110010011110000011101101000000010011000010000101110010101100100110100100010010100010100101011001000001000100010101110110110100001001000111011000101101001101111110000111010101000110010010111001000010100111110000001000010111000000100110011
