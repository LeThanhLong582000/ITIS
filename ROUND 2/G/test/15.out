69
207
386
302
52
467
129
318
458
407
108
122
97
327
282
100
340
253
118
217
204
71
245
105
74
450
109
311
393
95
75
234
455
260
429
248
232
356
326
197
476
114
194
183
376
198
232
422
396
327
382
77
145
399
282
122
169
